android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -406,7 +404,7 @@ constructor(
 
   /**
    * Add or update the [questionnaireResponse] resource with the passed content, and if an
-   * [actionParamUpdatableId] is set also update the resource it refers to by extracting its
+   * [actionParameterList] is set also update the resource it refers to by extracting its
```

Comment:
Can we reword this comment
```suggestion
   * [actionParameterList] represents the IDs of the resources to be updated on Questionnaire edit.
```